specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/virtualNetworkGateway.json

```diff
@@ -2071,6 +2071,15 @@
           "type": "boolean",
           "description": "ActiveActive flag."
         },
+        "enableDnsForwarding": {
```

Comment:
This is considered a breaking change and requires a new api version.

See section 'New property added to response'
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
`2019-08-01` isn't in master yet, we have a separate branch with new API version specifically because of that requirement

Reply:
Also the new property will not be serialized in the response for older versions of apis .  That's the standard NRP behavior implemented for these new properties as well.

Reply:
@ryansbenson gentle ping

Reply:
If this is in a new API version in the merge to master than this is an acceptable change.

Reply:
Anyway, it would be good (best practice) if these properties added at the end.